shared/profile/help.desktop.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+import React from 'react'
+import {Box, Text, Terminal, Icon, Avatar} from '../common-adapters'
+import {globalStyles, globalColors} from '../styles/style-guide'
+import {shell} from 'electron'
+
+type Props = {
+  username: string
+};
+
+const RenderHelp = ({username}: Props) => (
+  <Box style={stylesScrollContainer}>
+    <Box style={stylesContainer}>
+      <Box style={styleIconHeader}>
+        <Box style={styleIconWrapper}><Icon type='icon-fancy-bubbles-123-x-64' /></Box>
+        <Box style={styleIconWrapper}><Avatar size={48} username={username} /></Box>
+      </Box>
+      <Box style={styleTextHeader}>
+        <Text type='Body'>
+          Until this tab is ready, you must use <Text
+            type='BodyPrimaryLink'
+            onClick={() => shell.openExternal(`https://keybase.io/${username}`)}>
+            the website
+          </Text> to edit your bio, full name, and location.
+        </Text>
+      </Box>
+      <Box style={styleBody}>
+        <Text type='BodySmall' style={{...styleBodyText}}>
+          However, you can use the terminal to manage your keys and identity.
+          When you run the commands below, you’ll see your <Text
+            type='BodyPrimaryLink'
+            onClick={() => shell.openExternal(`https://keybase.io/${username}/graph`)}>
+            identity graph
+          </Text> update on the site.
+        </Text>
+        <Terminal style={styleTerminal}>
+          <Box style={{...styleTerminalGroups}}>
```

Comment:
a side effect of this column choice is that it'll wrap all messed up. i don't think we really need to fix this but you might want to set a minWidth on this container so it won't do that